buf.gen.yaml

```diff
@@ -0,0 +1,5 @@
+version: v1
+plugins:
+  - name: go
+    out: parser/testproto
```

Comment:
do we need this?

Reply:
It's necessary for proto parser unit testing. I haven't any ideas how do it more correctly.